test/framework/functional/output_cloudwatch.go

```diff
@@ -141,7 +142,7 @@ func (f *CollectorFunctionalFramework) GetLogGroupByType(inputName string) ([]st
 	found := false
 	for _, l := range logGroupsOutput.LogGroups {
 		// Filter by type and get all
-		if *l.LogGroupName == "group-prefix."+inputName {
+		if *l.LogGroupName == "group-prefix-"+inputName {
```

Comment:
why this chane?

Reply:
I guess the bigger question here is what characters are allowed/not allowed for this as well as indices for other outputs.